src/com/example/powerup/SessionHistory.java

```diff
@@ -0,0 +1,6 @@
+package com.example.powerup;
+
+public class SessionHistory {
+	public static Integer CurrQID = 1;
+	public static Integer CurrSessionID = 1;
```

Comment:
Use lowercase for variables. Make them private and use getters and setters if possible